Samples/Samples/View/LauncherPage.xaml

```diff
@@ -22,7 +22,7 @@
                     <ColumnDefinition Width="*" />
                     <ColumnDefinition Width="*" />
                 </Grid.ColumnDefinitions>
-                <Entry Grid.Row="0" Grid.ColumnSpan="2" Text="{Binding LaunchUri}" />
+                <Entry Grid.Row="0" Grid.ColumnSpan="2" Text="{Binding LaunchUri}" TextColor="Black" />
```

Comment:
These should no longer be needed. I just added a new style to fix this for iOS:
https://github.com/xamarin/Essentials/pull/1610

Reply:
We can roll back, I just didn't see the text I'm typing

Reply:
Yeah, thanks. We can fix this in a new PR so we don't have too much in this one.